demo-extended/src/main/java/com/novoda/downloadmanager/demo/extended/batches/Batch.java

```diff
@@ -0,0 +1,51 @@
+package com.novoda.downloadmanager.demo.extended.batches;
+
+import com.novoda.downloadmanager.lib.DownloadManager;
+
+public class Batch {
```

Comment:
They're just local model objects used to encapsulate the data from the db. One major difference between them is that `Download` has the path to the individual file as opposed to the `Batch` which doesn't.